File: test_archives/cpp_functions/batch5/function3.cpp
#include <iostream>
using namespace std;

class function3
{
public:
    // Initializing the Class
    function3(string name, int age, string gender, string address, int phone_number)
    {
        this->name = name;
        this->age = age;
        this->gender = gender;
        this->address = address;
        this->phone_number = phone_number;
        this->total_value = 0;
    }
    
    // Method to Calculate the Final Value 
    int calculate_value()
    {
        // Calculating the Final Value
        this->total_value = this->age + this->phone_number;
        return this->total_value;
    }
    
    // Class Variables
    string name;
    int age;
    string gender;
    string address;
    int phone_number;
    int total_value;
};

File: test_archives/py_functions/batch5/function3.py
class function3:
    # Initializing the Class
    def __init__(self, name, age, gender, address, phone_number):
        self.name = name 
        self.age = age 
        self.gender = gender 
        self.address = address 
        self.phone_number = phone_number 
        self.total_value = 0 
        
    # Method to Calculate the Final Value 
    def calculate_value(self):
        # Calculating the Final Value
        self.total_value = self.age + self.phone_number 
        return self.total_value